tools/topology/sof-cml-da7219-max98357a.m4

```diff
@@ -0,0 +1,238 @@
+#
+# Topology for Cometlake with da7219 headset on SSP0, max98357a spk on SSP1
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Cannonlake DSP configuration
+include(`platform/intel/cnl.m4')
+include(`platform/intel/dmic.m4')
+include(`abi.h')
+
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM0 ----> volume -----> SSP1   (speaker - maxim98357a, BE link 0)
+# PCM1 <---> volume <----> SSP0   (headset - da7219, BE link 1)
+# PCM2 <---- DMIC01 (dmic0 capture, BE link 2)
+# PCM5 ----> volume -----> iDisp1 (HDMI/DP playback, BE link 3)
+# PCM6 ----> volume -----> iDisp2 (HDMI/DP playback, BE link 4)
+# PCM7 ----> volume -----> iDisp3 (HDMI/DP playback, BE link 5)
+# PCM8 <---- volume <----- DMIC16k (dmic16k, BE link 6)
+#
+
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     frames, deadline, priority, core)
+
+# Low Latency playback pipeline 1 on PCM 1 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	1, 1, 2, s32le,
+	48, 1000, 0, 0)
```

Comment:
Has this been tested on master ? The PCM_ADD arguments have been updated, please check with other topologies on master.

Reply:
I have created a pull request for master also. Master only build check i have done. for cml-008-patch-backport its been tested on chromos device.

This commit i have cherry picked from cml-007-drop-stable branch. (https://github.com/thesofproject/sof/commit/05c8e9690dfb217232613f28d1c48424777f92ea)
Yes this commit does not include the PCM_ADD args change. For this i have added second commit in this pull request to add required arg changes.
(https://github.com/thesofproject/sof/pull/2315/commits/d18097084b53885ec8da049313f8bf59449c16ef)

Reply:
these should be squashed and rebased

Reply:
Agreed and done.